src/helper.ts

```diff
@@ -1,13 +1,12 @@
-import unpackaging from "../lib/unpack";
-import unpackagingWasm from "../lib/unpack.wasm";
+import unpack, { WasmModule } from "./unpack.js";
+import unpackWasm from "../src/unpack.wasm";
```

Comment:
```suggestion
import unpackWasm from "./unpack.wasm";
```
`unpack.wasm` should be placed to the `lib` directory by the build script, not the `src` directory